packages/components/src/internal/util/RouteLeave.tsx

```diff
@@ -17,27 +17,34 @@ export const ON_LEAVE_DIRTY_STATE_MESSAGE =
  * in the browser will not change. (Issue 39633).
  * See also https://stackoverflow.com/questions/32841757/detecting-user-leaving-page-with-react-router
  *
+ * TODO: Seems like there are some additional tools in newer versions of react-router:
+ *  https://stackoverflow.com/questions/62792342/in-react-router-v6-how-to-check-form-is-dirty-before-leaving-page-route
+ *
  * @param currentLocation the location of the current page
+ * @param event
+ * @param isUnload
```

Comment:
nit: `isUnload` is not a parameter to this method. I assume you had it one point and removed it. Additionally, there is a danglingly comma in the function signature.